definitions/ext-fibre_channel_switch/golden_metrics.yml

```diff
@@ -13,6 +14,7 @@ memoryUtilization:
     select: average(kentik.snmp.MemoryUtilization)
     from: Metric
     where: "provider = 'kentik-fibre-channel'"
+    eventId: entity.guid
```

Comment:
that is not really required, as it's the default value for the eventId

Reply:
It is for set same query values as the rest.